framework/include/executioners/Transient.h

```diff
@@ -167,12 +167,18 @@ class Transient : public Executioner
    * Return the start time
    * @return The start time
    */
-  Real getStartTime() { return _start_time; }
+  Real getStartTime() const { return _start_time; }
 
   /**
    * Get the end time
    * @return The end time
    */
+  Real getEndTime() const { return _end_time; }
```

Comment:
The different name is not necessary but perhaps it's useful to help elucidate that they are there for different intents

Reply:
yeah, that was my thinking. The current naming scheme seemed to consistently distinguish between the getters and writable reference returning functions.